lib/player.rb

```diff
@@ -0,0 +1,21 @@
+require "pry"
+
+class Player
+
+  @@positions = []
```

Comment:
It's really great to see you reach for these new concepts. I don't think this is the best use case for it, but glad to see you challenge yourself!